myuw/static/js/card/upass.js

```diff
@@ -0,0 +1,36 @@
+var UPassCard = {
+    name: 'UPassCard',
+    dom_target: undefined,
+
+    render_init: function () {
+        UPassCard.dom_target = $('#UPassCard');
+        WSData.fetch_upass_data(UPassCard.render_upon_data, UPassCard.render_error);
+    },
+
+    render_error: function (status) {
+        if (status === 404) {
+            UPassCard.dom_target.hide();
```

Comment:
Is hiding on 404 the appropriate behavior?  From what I saw the upass API shouldn't return a 404 so it might be valuable to show the error card when the API does 404.